### PR TITLE
Improved image uploading feature

### LINKED ISSUE #27: User/master avatar upload/download integration
@xmasterpiecex You can now integrate avatar downloads and uploads
There are two endpoints for this `GET /Asset/profile/get/{imageId}` and `POST /Asset/profile/upload`, for details use [swagger UI](https://github.com/Whirlcode/dobirna-gra/wiki/Wiki)

Also you should not use the current URL, where the client is hosted, but use the server URL, for this you can use `import.meta.env.VITE_API`.

### PR DESCRIPTION
**Pull request by xmasterpiecex**

### CODE REVIEW

File: src/components/entryComponents/UploadImgButton.tsx [hunk 1]
```diff
@@ -12,14 +12,22 @@ const VisuallyHiddenInput = styled("input")`
 `;
 
 export default function UploadImgButton() {
+  const me = useAppSelector(s => s.gameState.me)
   const [userImg, setUserImg] = useState<string>(
-    "https://placehold.co/600x400?text=Choose+your+picture"
-  );
+    'https://placehold.co/600x400/263238/48af63?text=Upload+Image');
```
Comment (uspiritwolf) [inline src/components/entryComponents/UploadImgButton.tsx]: I think we should rely less on third party service to make the app more stable and independent of other sites

File: src/components/gameScene/playerComp/PlayerCard.tsx [hunk 2]
```diff
@@ -12,21 +12,23 @@ import {
   Typography,
 } from "@mui/joy";
 import { useEffect, useRef, useState } from "react";
-import defImg from '@app/assets/maxresdefault.jpg'
 import hubController from "@app/SignalR/HubController";
+import defImg from '@app/assets/maxresdefault.jpg'
```
Comment (uspiritwolf) [inline src/components/gameScene/playerComp/PlayerCard.tsx]: Please rename the file to something more readable and appropriate.

Example: DefaultUserImage.png

File: src/components/gameScene/playerComp/PlayerCard.tsx [hunk 3]
```diff
@@ -12,21 +12,23 @@ import {
...
 
 export default function PlayerCard({
   initialVal,
   step,
   playerName,
   scoreOfPlace,
-  indexOfPlace
+  indexOfPlace,
+  imgId
```
Comment (uspiritwolf) [inline src/components/gameScene/playerComp/PlayerCard.tsx:33]: Is it possible to just pass the PlayerPlaceData type instead of "destructuring" it?

**Review by uspiritwolf — CHANGES_REQUESTED** [review]

File: src/components/gameScene/playerComp/PlayerCard.tsx (hunk 2 above)
Comment (xmasterpiecex) [reply src/components/gameScene/playerComp/PlayerCard.tsx]: Good point

File: src/components/gameScene/playerComp/PlayerCard.tsx (hunk 3 above)
Comment (xmasterpiecex) [reply src/components/gameScene/playerComp/PlayerCard.tsx:33]: Yep, I've created type "TPlayerCardData".

File: src/components/entryComponents/UploadImgButton.tsx (hunk 1 above)
Comment (xmasterpiecex) [reply src/components/entryComponents/UploadImgButton.tsx]: Ok, changed it to static asset image

**Review by uspiritwolf — APPROVED** [review]

**Pull request merged**